### PR TITLE
BC-7824 add data testid for classes

### PR DESCRIPTION
**Pull request by atlldwp**
# Description

## Links to Tickets or other pull requests
BC-7824

## Changes
- Added the data-testids on the class related page elements.

## Approval for review

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [ ] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/administration/forms/form-classes-edit.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<input type="hidden" name="referrer" value="{{../referrer}}" />
 
 	<a class="d-block mb-1" href="/administration/classes/{{../class/_id}}/manage"
-		title="Klasse '{{../class.name}}' verwalten.">
+		title="Klasse '{{../class.name}}' verwalten." data-testid="go-to-manage-class-btn">
```
Comment (MartinSchuhmacher) [inline views/administration/forms/form-classes-edit.hbs:8]: "go-to" does not tell much, if you really wana have something more specific I would go with `manage-class-btn`

File: views/administration/forms/form-classes-edit.hbs [hunk 2]
```diff
@@ -112,7 +112,7 @@
 		{{/userHasPermission}}
 	{{/ifeq}}
 	{{#ifeq mode "edit"}}
-		<button class="btn btn-primary" type="submit" disabled>{{$t "global.button.saveChanges" }}</button>
+		<button data-testId="confirmClassEdit" class="btn btn-primary" type="submit" disabled>{{$t "global.button.saveChanges" }}</button>
```
Comment (MartinSchuhmacher) [inline views/administration/forms/form-classes-edit.hbs:115]: `confirm-class-edit`

File: views/administration/forms/form-classes-manage.hbs [hunk 3]
```diff
@@ -6,13 +6,13 @@
     <div class="row">
         <a class="d-block mb-1"
             href="/administration/classes/{{@root.class/_id}}/edit"
-            title="{{$t "administration.classes.placeholer.change" (dict "name" @root.class.name)}}">
+            title="{{$t "administration.classes.placeholer.change" (dict "name" @root.class.name)}}" data-testid="go-to-rename-class-btn">
```
Comment (MartinSchuhmacher) [inline views/administration/forms/form-classes-manage.hbs:9]: same here as above regarding "go-to"

File: views/administration/forms/form-classes-edit.hbs (hunk 1 above)
Comment (atlldwp) [reply views/administration/forms/form-classes-edit.hbs]: Done!

File: views/administration/forms/form-classes-edit.hbs (hunk 2 above)
Comment (atlldwp) [reply views/administration/forms/form-classes-edit.hbs]: Done!

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**